src/main/groovy/wooga/gradle/build/unity/UnityBuildPlugin.groovy

```diff
@@ -112,40 +116,32 @@ class UnityBuildPlugin implements Plugin<Project> {
-                        if (path.contains("plugins")
-                                && !((name == "plugins") || (name == "plugins.meta"))) {
-                            /*
-                             Why can we use / here? Because {@code element} is a {@code FileTreeElement} object.
-                             The getPath() method is not the same as {@code File.getPath()}
-                             From the docs:
-
-                             * Returns the path of this file, relative to the root of the containing file tree. Always uses '/' as the hierarchy
-                             * separator, regardless of platform file separator. Same as calling <code>getRelativePath().getPathString()</code>.
-                             *
-                             * @return The path. Never returns null.
-                             */
-                            if (t.getBuildPlatform()) {
-                                status = path.contains("plugins/" + t.getBuildPlatform().get())
-                            } else {
-                                status = true
-                            }
+                def includeSpec = { FileTreeElement element ->
+                    def path = element.getRelativePath().getPathString().toLowerCase()
+                    def name = element.name.toLowerCase()
+                    def status = true
+                    if (path.contains("plugins") && !((name == "plugins") || (name == "plugins.meta"))) {
+                        /*
+                         Why can we use / here? Because {@code element} is a {@code FileTreeElement} object.
+                         The getPath() method is not the same as {@code File.getPath()}
+                         From the docs:
+
+                         * Returns the path of this file, relative to the root of the containing file tree. Always uses '/' as the hierarchy
+                         * separator, regardless of platform file separator. Same as calling <code>getRelativePath().getPathString()</code>.
+                         *
+                         * @return The path. Never returns null.
+                         */
+                        if (t.getBuildPlatform()) {
+                            status = path.contains("plugins/" + t.getBuildPlatform().get())
+                        } else {
+                            status = true
                         }
-
-                        status
                     }
+                    return status
                 }
 
-                def excludeSpec = new Spec<FileTreeElement>() {
-                    @Override
-                    boolean isSatisfiedBy(FileTreeElement element) {
-                        return extension.ignoreFilesForExportUpToDateCheck.contains(element.getFile())
-                    }
+                def excludeSpec = { FileTreeElement element  ->
+                    return extension.ignoreFilesForExportUpToDateCheck.contains(element.getFile())
```

Comment:
I see the reason for the change but wouldn't have done it in this patch to be honest. But leave it.